### DISCUSSION TITLE
Implement OTP element

### DISCUSSION THREAD

**Discussion by adamberecz** [vueform/vueform#64] [Ideas]
Implement OTP element

**Comment by adamberecz** [comment]
![otp](https://github.com/vueform/vueform/assets/16024069/1d808233-e8cb-4f03-be16-515f18616094)

**Comment by digino** [comment]
Hello Adam
Hope you're fine?
Just know if you have started working or plan to work on it

**Reply by adamberecz** [reply]
Unfortunately I haven't really got there yet and it's currently not on the close priorities.

**Reply by digino** [reply]
Okay
I'll try to work on it when I have free time and submit it to you when appropriate.
Regards
